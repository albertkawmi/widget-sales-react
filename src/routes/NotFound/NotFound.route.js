import React from 'react'

const NotFound = () => (
  <h2>Oops! The page you requested doesn't exist</h2>
)

export default NotFound
