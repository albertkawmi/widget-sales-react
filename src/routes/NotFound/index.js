import NotFound from './NotFound.route'
export default NotFound
