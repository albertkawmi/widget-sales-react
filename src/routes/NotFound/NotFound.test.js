/* eslint-env jest */
import React from 'react'
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import NotFound from './NotFound.route'

Enzyme.configure({ adapter: new Adapter() })

const wrapper = Enzyme.shallow(<NotFound />)

describe('<NotFound />', () => {
  it('renders a 404 page', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
