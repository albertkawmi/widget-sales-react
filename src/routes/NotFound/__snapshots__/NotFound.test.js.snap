// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> renders a 404 page 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2
          className="page-heading"
>
          Oops! The page you requested doesn't exist
</h2>,
        <p>
          You can 
          <a
                    href="/"
          >
                    go back to the homepage
          </a>
           or, if you like code, 
          <a
                    href="https://github.com/albertkawmi/widget-sales-react"
          >
                    head to GitHub
          </a>
           to find out more about this app.
</p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Oops! The page you requested doesn't exist",
          "className": "page-heading",
        },
        "ref": null,
        "rendered": "Oops! The page you requested doesn't exist",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "You can ",
            <a
              href="/"
>
              go back to the homepage
</a>,
            " or, if you like code, ",
            <a
              href="https://github.com/albertkawmi/widget-sales-react"
>
              head to GitHub
</a>,
            " to find out more about this app.",
          ],
        },
        "ref": null,
        "rendered": Array [
          "You can ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "go back to the homepage",
              "href": "/",
            },
            "ref": null,
            "rendered": "go back to the homepage",
            "type": "a",
          },
          " or, if you like code, ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "head to GitHub",
              "href": "https://github.com/albertkawmi/widget-sales-react",
            },
            "ref": null,
            "rendered": "head to GitHub",
            "type": "a",
          },
          " to find out more about this app.",
        ],
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2
            className="page-heading"
>
            Oops! The page you requested doesn't exist
</h2>,
          <p>
            You can 
            <a
                        href="/"
            >
                        go back to the homepage
            </a>
             or, if you like code, 
            <a
                        href="https://github.com/albertkawmi/widget-sales-react"
            >
                        head to GitHub
            </a>
             to find out more about this app.
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Oops! The page you requested doesn't exist",
            "className": "page-heading",
          },
          "ref": null,
          "rendered": "Oops! The page you requested doesn't exist",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "You can ",
              <a
                href="/"
>
                go back to the homepage
</a>,
              " or, if you like code, ",
              <a
                href="https://github.com/albertkawmi/widget-sales-react"
>
                head to GitHub
</a>,
              " to find out more about this app.",
            ],
          },
          "ref": null,
          "rendered": Array [
            "You can ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "go back to the homepage",
                "href": "/",
              },
              "ref": null,
              "rendered": "go back to the homepage",
              "type": "a",
            },
            " or, if you like code, ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "head to GitHub",
                "href": "https://github.com/albertkawmi/widget-sales-react",
              },
              "ref": null,
              "rendered": "head to GitHub",
              "type": "a",
            },
            " to find out more about this app.",
          ],
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
